model-archiver/setup.py

```diff
@@ -45,12 +45,35 @@ def detect_model_archiver_version():
     return model_archiver.__version__.strip() + 'b' + str(date.today()).replace('-', '')
 
 
+def get_nightly_version():
+    today = date.today()
+    return today.strftime("%Y.%m.%d")
+
+
+
+
+    version = get_nightly_version() if is_nightly else detect_workflow_archiver_version()
```

Comment:
Nit: Copy/paste error?

Reply:
Not a nit at all, thank you for finding this